frontend/ui/impl/build.gradle

```diff
@@ -6,15 +6,9 @@ dependencies {
   implementation project(':binding')
   implementation project(':branchLayout:api')
   implementation project(':backend:api')
-  implementation project(':frontend:compatibilityBridge')
-  implementation project(':frontend:defs')
-  implementation project(':frontend:dataKeys')
   implementation project(':frontend:graph:api')
-  implementation project(':frontend:vfsUtils')
   implementation project(':frontend:file')
-  implementation project(':frontend:providerServices')
```

Comment:
If it comes to `frontend:providerServices`... I'd rather see this module's content moved into `frontend:ui:api` (as these services logicaly constitute part of the API for UI)... would it be possible without creating a cyclic dependency?

Reply:
that is, in practice to remove providerServices module (and content from them put into frontend:ui:api module )?